terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -230,10 +230,8 @@
 ]
 
 other_public_keys = {
-    'dev': operator_keys,
-    'anvildev': operator_keys,
-    'anvilprod': operator_keys,
-    'prod': []
+    name: [] if config.Deployment(name).is_stable else operator_keys
+    for name in ('dev', 'anvildev', 'anvilprod', 'prod')
```

Comment:
Avoid hard-coding all deployment names. Note that only the current deployment is actually used as a key into this dictionary.